engines/sludge/keymapper_tables.h

```diff
@@ -0,0 +1,556 @@
+#include "backends/keymapper/action.h"
+#include "backends/keymapper/keymapper.h"
+#include "backends/keymapper/standard-actions.h"
+
+#include "common/translation.h"
+
+namespace Sludge {
+inline Common::KeymapArray getSludgeKeymaps(const char *target, const Common::String &gameId, const char *gameDescr) {
+
+    using namespace Common;
+
+    Keymap *gameKeyMap = new Keymap(Keymap::kKeymapTypeGame, gameId, gameDescr);
```

Comment:
Why is the keymap name set to the gameId? Most engines tend to have an engine specific keymap usually named with the format "{engine}-default" and a game specific keymap usually named "game-shortcuts".

Reply:
I should mention that Wintermute does implement it in a similar way to this PR. I'm not aware of the reasoning behind it there either. It may be correct, but I'd like to understand better.